pallet/ethy/src/lib.rs

```diff
@@ -570,7 +570,7 @@ pub mod pallet {
 
 			// 3) Try process delayed proofs
 			consumed_weight += DbWeight::get().reads(2u64);
-			if PendingEventProofs::<T>::iter().next().is_some() && !BridgePaused::<T>::get() {
+			if !Self::bridge_paused() && PendingEventProofs::<T>::iter().next().is_some() {
```

Comment:
nit: to remove level of nesting

```suggestion
			if Self::bridge_paused() || PendingEventProofs::<T>::iter().next().is_none() {
                return consumed_weight;
            }
            ...
```

Reply:
Leaving as is because this whole piece of code is refactored in #831